pubspec.yaml

```diff
@@ -19,4 +19,4 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  very_good_analysis: ^2.4.0
+  very_good_analysis: ^3.0.2
```

Comment:
can we also update the version used in `analysis_options.yaml`?

Reply:
done